flagai/model/base_model.py

```diff
@@ -47,23 +47,67 @@ def from_pretrain(cls,
                       device="cpu",
                       **kwargs):
         model_id = None
+
```

Comment:
add comments 
add parameters for load_local
add assert for config_path & checkpoint_path